### PR TITLE
docs: announcing Rsbuild 1.0 🎉

### PR DESCRIPTION
**Pull request by chenjiahan**
## Summary

Announcing Rsbuild 1.0 🎉

## Checklist

- [ ] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: website/docs/en/community/releases/v1-0.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+As the ecosystem evolved, full-stack web frameworks such as Next.js, Nuxt, and Remix became popular; Vite was introduced as a lightweight build tool and also gained popularity. However, CRA and Vue CLI gradually stopped getting maintained.
+
+When we look at the npm download numbers for webpack, CRA, and Vue CLI, we find that a large number of projects are still using these tools. For example, webpack has about 25 million weekly downloads, and CRA has nearly 3 million weekly downloads. Many of these projects are CSR applications that do not require the SSR features of full-stack frameworks. Vite seems like a good choice, but after using Vite in our ByteDance projects, we found that the migrating from webpack to Vite comes with high costs and introduces new problems, such as dev and build inconsistency, and slow page refreshes in large applications during development.
```
Comment (Romej) [inline website/docs/en/community/releases/v1-0.mdx:25]:
```suggestion
When we look at the npm download numbers for webpack, CRA, and Vue CLI, we find that a large number of projects are still using these tools. For example, webpack has about 25 million weekly downloads, and CRA has nearly 3 million weekly downloads. Many of these projects are CSR applications that do not require the SSR features of full-stack frameworks. Vite seems like a good choice, but after using Vite in our ByteDance projects, we found that the migration from webpack to Vite comes with high costs and introduces new problems, such as dev and build inconsistency, and slow page refreshes in large applications during development.
```

File: website/docs/en/community/releases/v1-0.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+For a long time, developers using webpack have been bothered by two major issues: **slow build times and configuration complexity**.
+
+We have used Rust to rewrite webpack into [Rspack](https://github.com/web-infra-dev/rspack), which addresses the slow build issue. However, to maintain compatibility with the webpack ecosystem, Rspack retains webpack's configuration and API, which means it still has some complexity and a learning curve.
+
+### Evolution of the ecosystem
+
+In the early days, there were some excellent tools within the webpack ecosystem, such as Create React App (CRA) and Vue CLI. These tools provided best practices for building React or Vue applications, while hiding the complex webpack configuration. As a result, many React and Vue users used these tools to build applications without having to configure webpack from scratch.
+
+As the ecosystem evolved, full-stack web frameworks such as Next.js, Nuxt, and Remix became popular; Vite was introduced as a lightweight build tool and also gained popularity. However, CRA and Vue CLI gradually stopped getting maintained.
```
Comment (Romej) [inline website/docs/en/community/releases/v1-0.mdx:23]:
```suggestion
As the ecosystem evolved, full-stack web frameworks such as Next.js, Nuxt, and Remix became popular; Vite was introduced as a lightweight build tool and also gained popularity. However, CRA and Vue CLI gradually stopped being maintained.
```

File: website/docs/en/community/releases/v1-0.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+For the webpack ecosystem, we discovered a sad fact: **the webpack ecosystem lacks a build tool that is easy to use and well maintained**. The tool should be as user-friendly as CRA and Vue CLI, fully meet the needs of CSR application development, and have features such as fast startup and plugin support similar to Vite.
+
+### The birth of Rsbuild
+
+During the development of Rspack, we became aware of the above problems and decided to create a modern build tool based on Rspack called **Rsbuild**.
+
+![](https://assets.rspack.dev/rsbuild/assets/rsbuild-1-0-homepage.png)
+
+Rsbuild is built on top of Rspack. We designed Rsbuild with an easy-to-use, TypeScript-friendly API and a set of carefully designed configuration to fully leverage the Rspack's build performance while reducing configuration complexity and high up-front costs.
```
Comment (Romej) [inline website/docs/en/community/releases/v1-0.mdx:35]:
```suggestion
Rsbuild is built on top of Rspack. We designed Rsbuild with an easy-to-use, TypeScript-friendly API and a set of carefully designed configurations to fully leverage the Rspack's build performance while reducing configuration complexity and high up-front costs.
```

File: website/docs/en/community/releases/v1-0.mdx (hunk 1 above)
Comment (chenjiahan) [reply website/docs/en/community/releases/v1-0.mdx:25]: How about removing `the`?

image

https://www.deepl.com/en/write#en/en-us/Vite%20seems%20like%20a%20good%20choice%2C%20but%20after%20using%20Vite%20in%20our%20ByteDance%20projects%2C%20we%20found%20that%20the%20migration%20from%20webpack%20to%20Vite%20comes%20with%20high%20costs%20and%20introduces%20new%20problems%2C%20such%20as%20dev%20and%20build%20inconsistency%2C%20and%20slow%20page%20refreshes%20in%20large%20applications%20during%20development.

**Comment by chenjiahan** [comment]
Thanks for your review @Romej ❤️

File: website/docs/en/community/releases/v1-0.mdx (hunk 1 above)
Comment (Romej) [reply website/docs/en/community/releases/v1-0.mdx:25]: Yes that also works !

File: website/docs/en/community/releases/v1-0.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+> September 10, 2024
+
+![](https://assets.rspack.dev/rsbuild/assets/rsbuild-1-0-banner.png)
+
+We are pleased to announce the release of Rsbuild 1.0!
+
+## Why Rsbuild
+
+For a long time, developers using webpack have been bothered by two major issues: **slow build times and configuration complexity**.
```
[the remark this answers is not in the document]
Comment (chenjiahan) [reply website/docs/en/community/releases/v1-0.mdx:15]: no need to separate them

File: website/docs/zh/community/releases/v1-0.mdx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+我们使用 Rust 将 webpack 重写为 [Rspack](https://github.com/web-infra-dev/rspack)，解决了构建慢的问题。但为了兼容 webpack 生态，Rspack 保留了 webpack 的配置和 API，这意味着它依然存在一定的复杂度和学习成本。
+
+### 生态的发展
+
+在早期，webpack 生态中出现了一些优秀的工具，比如 Create React App（简称 CRA）和 Vue CLI，它们为 React 或 Vue 应用提供最佳实践，隐藏了复杂的 webpack 配置。因此，许多 React 和 Vue 用户使用这些工具来创建应用，不需要从零开始配置 webpack。
+
+随着生态的发展，Next.js、Nuxt 和 Remix 等全栈 web 框架变得流行；Vite 推出后，作为一个轻量化的构建工具，也受到了众多开发者的青睐。而 CRA、Vue CLI 则是逐渐停止了维护。
+
+当我们查看 webpack、CRA 和 Vue CLI 的 npm 下载量时，会发现仍然有大量项目在使用这些工具。例如，webpack 有约 2500 万的周下载量，CRA 有近 300 万的周下载量。这些项目有很多是 CSR 应用，不需要使用全栈框架的 SSR 等特性；Vite 看起来是一个不错的选择，但我们在字节跳动的项目中实践后发现，从 webpack 迁移到 Vite 存在很高的成本，并且迁移带来了一些新问题，例如开发环境与生产环境的构建产物不一致、大型应用在开发过程中页面刷新缓慢等问题。
```
[the remark this answers is not in the document]
Comment (chenjiahan) [reply website/docs/zh/community/releases/v1-0.mdx:25]: I think use `webpack` here is ok, there are many webpack-based CSR apps

**Review by 9aoy — APPROVED** [review]

**Pull request merged**